AdyenCard/Utilities/CardType/CardBrandProvider.swift

```diff
@@ -26,7 +26,7 @@ internal struct CardBrandProviderParameters {
 /// Provide cardType detection based on BinLookup API.
 internal final class CardBrandProvider: AnyCardBrandProvider {
     
-    private static let minBinLength = 6
+    private static let minBinLength = 10
```

Comment:
according to Nicholas its 11 actually

Reply:
Yep. We do `bin > minBinLength ` :D

Reply:
minimum means its an allowed value, so if the minimum is 10 then it should be `if bin >= minBinLength ` :-D

Reply:
👍